robot-core/src/main/java/org/obolibrary/robot/export/Table.java

```diff
@@ -332,6 +332,20 @@ public String toHTML(String split, boolean standalone, boolean includeJS) {
     return sb.toString();
   }
 
+  public String toHTMLList() {
+    StringBuilder sb = new StringBuilder();
+    sb.append("<head>\n")
+        .append(
+            "\t<link rel=\"stylesheet\" href=\"https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css\">\n")
```

Comment:
I'd like all these versioned links factored out to shared variables. This Bootstrap version doesn't match others in this same file.